src/ng/directive/ngModel.js

```diff
@@ -1065,9 +1072,41 @@ var DEFAULT_REGEXP = /(\s+|^)default(\s+|$)/;
  * @name ngModelOptions
  *
  * @description
- * Allows tuning how model updates are done. Using `ngModelOptions` you can specify a custom list of
- * events that will trigger a model update and/or a debouncing delay so that the actual update only
- * takes place when a timer expires; this timer will be reset after another change takes place.
+ * This directive allows you to modify the behaviour of ngModel and input directives within your
+ * application. You can specify an ngModelOptions directive on any element and the settings affect
+ * the ngModel and input directives on all descendent elements.
+ *
+ * The ngModelOptions settings are found by evaluating the value of the ngModelOptions attribute as
+ * an Angular expression. This expression should evaluate to an object, whose properties contain
+ * the settings.
+ *
+ * If a setting is not specified as a property on the object for a particular ngModelOptions directive
+ * then it will inherit that setting from the first ngModelOptions directive found by traversing up the
+ * DOM tree.
+ *
+ * For example in the following fragment of HTML ...
+ *
+ *
+ * ```html
+ * <div ng-model-options="{ allowInvalid: true }">
+ *   <form ng-model-options="{ updateOn: \'blur\' }">
+ *     <input ng-model-options="{ updateOn: \'default\' }">
+ *   </form>
+ * </div>
+ * ```
+ *
+ * ... the `input` element will effective have the follow settings ...
```

Comment:
Not a big deal, but shouldn't `effective` be `effectively` ?
(I am not insisting or anything, just bringing it up again in case you overlooked it; you obviously know better (being a native speaker :smiley:).)